frontend/src/v5/ui/routes/dashboard/projects/containers/containersList/containersList.helpers.ts

```diff
@@ -15,54 +15,69 @@
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+import { useParams } from 'react-router';
 import { IContainer } from '@/v5/store/containers/containers.types';
 import { formatMessage } from '@/v5/services/intl';
+import { ContainersActionsDispatchers } from '@/v5/services/actionsDispatchers/containersActions.dispatchers';
+import { DialogsActions } from '@/v5/store/dialogs/dialogs.redux';
+import { useDispatch } from 'react-redux';
 
-export const getContainerMenuItems = (id: IContainer['_id']) => [
-	{
-		key: 1,
-		title: formatMessage({ id: 'containers.ellipsisMenu.loadContainer', defaultMessage: 'Load Container in 3D Viewer' }),
-		to: `/${id}`,
-	},
-	{
-		key: 2,
-		title: formatMessage({ id: 'containers.ellipsisMenu.uploadNewRevision', defaultMessage: 'Upload new Revision' }),
-		onClick: () => { },
-	},
-	{
-		key: 3,
-		title: formatMessage({ id: 'containers.ellipsisMenu.viewIssues', defaultMessage: 'View Issues' }),
-		onClick: () => { },
-	},
-	{
-		key: 4,
-		title: formatMessage({ id: 'containers.ellipsisMenu.viewRisks', defaultMessage: 'View Risks' }),
-		onClick: () => { },
-	},
-	{
-		key: 5,
-		title: formatMessage({ id: 'containers.ellipsisMenu.viewRevisions', defaultMessage: 'View Revisions' }),
-		onClick: () => { },
-	},
-	{
-		key: 6,
-		title: formatMessage({ id: 'containers.ellipsisMenu.editPermissions', defaultMessage: 'Edit Permissions' }),
-		onClick: () => { },
-	},
-	{
-		key: 7,
-		title: formatMessage({ id: 'containers.ellipsisMenu.shareContainer', defaultMessage: 'Share Container' }),
-		onClick: () => { },
-	},
-	{
-		key: 8,
-		title: formatMessage({ id: 'containers.ellipsisMenu.settings', defaultMessage: 'Settings' }),
-		onClick: () => {
+export const getContainerMenuItems = (id: IContainer['_id'], name: IContainer['name']) => {
```

Comment:
Instead of passing indiviual properties, lets pass the whole container object. TBH I dont like this `getContainerMenuItems(..)` mechanism. I think I would prefer to have something like:

```
const ContainerEllipsisMenu =  (container:IContainer) => {
<EllipsisMenu>
    <EllipsisMenuItem onClick="...">
         <FormattedMessage id="containers.ellipsisMenu.loadContainer" defaultMessage="Load Container in 3D Viewer" />
    <EllipsisMenuItem />
         ....  
  <EllipsisMenu>
...
```

Lets not change it for now, but I might make a new issue for this in the future.